src/Mutation/MediaItemCreate.php

```diff
@@ -253,10 +253,15 @@ public static function mutate_and_get_payload() {
 			 * post_status (inherit if not entered)
 			 * post_mime_type (pulled from the file if not entered in the mutation)
 			 */
-			$attachment_id = wp_insert_attachment( $media_item_args, $file['file'], $attachment_parent_id );
+			$attachment_id = wp_insert_attachment( $media_item_args, $file['file'], $attachment_parent_id, true );
 
-			if ( 0 === $attachment_id || is_wp_error( $attachment_id ) ) {
-				throw new UserError( __( 'The Media Item failed to create', 'wp-graphql' ) );
+			if ( is_wp_error( $attachment_id ) ) {
```

Comment:
the `0 === $attachment_id` should be added back: https://developer.wordpress.org/reference/functions/wp_insert_attachment/